static/app/views/issueList/overview.tsx

```diff
@@ -1023,6 +1028,58 @@ class IssueListOverview extends React.Component<Props, State> {
     this.fetchData(true);
   };
 
+  onUndo = () => {
+    const {organization, selection} = this.props;
+    const {actionTakenGroupData} = this.state;
+    const query = this.getQuery();
+
+    const groupIds = actionTakenGroupData.map(data => data.id);
+    const projectIds = selection?.projects?.map(p => p.toString());
+    const isMarkedReviewed = actionTakenGroupData.every(data => data.inbox);
+    const endpoint = `/organizations/${organization.slug}/issues/`;
+    // if action taken on Ignored tab, revert back to ignored instead of unresolved
+    const status = query.includes('is:ignored') ? 'ignored' : 'unresolved';
```

Comment:
This seems slightly confusing since we wouldn't be able to put it back into the right "ignored" group. Like if they had selected "ignore until this happens 100 times in one day"